### PR TITLE
New Footer

### PR DESCRIPTION
**Pull request by KimAT25**
Created new footer;
Full UI with responsive according to break points

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -6,4 +6,7 @@
 <app-lower-nav-bar></app-lower-nav-bar>
 <div class="app-container">
   <router-outlet></router-outlet>
+<!--  <footer>-->
+<!--    <app-new-footer></app-new-footer>-->
+<!--  </footer>-->
```
Comment (RomanDeveloperAcc) [inline src/app/app.component.html:11]: Please, delete commented code if it is not necessary

File: src/app/component/core/new-footer/new-footer.component.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import {footerIcons} from '../../../../assets/img/icon/footer/footer-icons';
+
+@Component({
+  selector: 'app-new-footer',
+  templateUrl: './new-footer.component.html',
+  styleUrls: ['./new-footer.component.scss']
+})
+export class NewFooterComponent implements OnInit {
+  public actualYear = new Date().getFullYear();
+  private fbIcon = footerIcons;
```
Comment (nabermi2020) [inline src/app/component/core/new-footer/new-footer.component.ts:11]: Please use one variable

Comment (nabermi2020) [reply src/app/component/core/new-footer/new-footer.component.ts:11]: And delete duplicates

Comment (KimAT25) [reply src/app/component/core/new-footer/new-footer.component.ts:11]: Fixed

**Pull request merged**